recipes/tcsh/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -eu -o pipefail
+
+ldconfig -v
+
+./configure 
+make
```

Comment:
Is there `make check` or similar that we can run?

Reply:
There is indeed...now added...